stable/consul/Chart.yaml

```diff
@@ -1,14 +1,12 @@
 apiVersion: v1
 name: consul
 home: https://github.com/hashicorp/consul
-version: 3.9.4
-appVersion: 1.5.3
-description: Highly available and distributed service discovery and key-value store
+version: 3.10.0
+appVersion: 1.6.2
+deprecated: true
+description: DEPRECATED Highly available and distributed service discovery and key-value store
```

Comment:
Please see https://github.com/helm/charts/blob/master/PROCESSES.md for deprecating a chart